en/docs/consuming-services/share-applications.md

```diff
@@ -1,17 +1,25 @@
-# Share Applications
+# Share applications
 
-Choreo's application-sharing feature allows you to share your applications with members within your organization. This facilitates collaborative efforts when there is a need for multiple members to work on the same application.
+Choreo's application-sharing feature allows you to share your applications with members within your organization. This promotes collaboration when multiple members need to work on the same application.
 
-Follow the steps given below to share an application with members of your organization:
+Follow these steps to share an application with members of your organization:
 
-1. Sign in to the [Choreo Developer Portal](https://devportal.choreo.dev/) and click the **Applications** tab.
-2. On the **Applications** page, click on the application you want to share.
-3. On the **Application Overview** page that opens, click the **Share** button displayed on the right side.
-4. In the **Share Application** dialog, enter one or more email addresses depending on the members with whom you want to share the applications.
+1. **Sign in to the Choreo Developer Portal**  
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. Any reason to do this? See https://developers.google.com/style/text-formatting and https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines.